src/components/routes/AppRoutes.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from "react";
+import { BrowserRouter, Link, Route, Switch } from "react-router-dom";
```

Comment:
Link not used

Reply:
We may want to add `noUnusedLocals` and `noUnusedParameters` to our tsconfig to prevent unused imports/variables.